tracing-subscriber/src/reload.rs

```diff
@@ -140,22 +211,25 @@ impl<S> Subscriber<S> {
 
 // ===== impl Handle =====
 
-impl<S> Handle<S> {
-    /// Replace the current subscriber with the provided `new_subscriber`.
+impl<T> Handle<T> {
+    /// Replace the current subscriber or filter with the provided `new_object`.
     ///
-    /// **Warning:** The [`Filtered`](crate::filter::Filtered) type currently can't be changed
+    /// **Warning:** If this `Handle` is being used with the [`Filtered`](crate::filter::Filtered)
+    /// subscriber, it currently can't be changed
     /// at runtime via the [`Handle::reload`] method.
     /// Use the [`Handle::modify`] method to change the filter instead.
```

Comment:
```suggestion
    /// **Warning:** [`Handle::reload`] cannot be used with the [`Filtered`](crate::filter::Filtered)
    /// subscriber; use [`Handle::modify`] instead (this [this issue] for additional details).
```

Reply:
I feel like I should still mention wrapping the filter?